lib/roadie/document.rb

```diff
@@ -50,14 +52,14 @@ def initialize(html)
       @html = html
       @asset_providers = ProviderList.wrap(FilesystemProvider.new)
       @external_asset_providers = ProviderList.empty
-      @css = ""
+      @css = +""
       @mode = :html
     end
 
     # Append additional CSS to the document's internal stylesheet.
     # @param [String] new_css
     def add_css(new_css)
-      @css << "\n\n" << new_css
+      @css += "\n\n" + new_css
```

Comment:
Shouldn't `<<` work since `@css` is initialized with a mutable string?

Reply:
This was a good point, I reworked my other changes to go back to `<<`